### PR TITLE
Fix spelling in main printer_mods/README.md for TipTophat

### PR DESCRIPTION
**Pull request by waytotheweb**
* [ x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [ x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [x ] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

This is to fix a small spelling mistake by me on my submission for the TipTophat in the main voron
printer_mods/README.txt

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ like so:
 ||[PITFT50 45-degree mount](./CannedBass/PITFT50_45_degree_mount)| Mount that puts the screen at a 45-degree angle for more vertical clearance |:x:|:grey_question:|:heavy_check_mark:|:x:|:heavy_check_mark:|
 | Ch4rlesB | [V0 Handles Slim](./Ch4rlesB/V0_Handles_Slim) | Simple and slim handles for the V0 | :heavy_check_mark: | :x: | :x: | :x: |
 | ChenTheDesignMaker | [Z Belt Clip](./ChenTheDesignMaker/Z_Belt_Clip) | Simple clip to keep extra Z belts clean and away from panels | :x: | :x: | :heavy_check_mark: | :x: | :x:|
-| chirpy | [V0.1 Raised Tophat](./chirpy/TipTophat) | Alternative Tophat design that uses stock panels and raises height be 30mm| :heavy_check_mark: | :x: | :x: | :x: |
+| chirpy | [V0.1 Raised Tophat](./chirpy/TipTophat) | Alternative Tophat design that uses stock panels and raises height by 30mm| :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (newphreak) [inline printer_mods/README.md:92]: Add one more ` :x: |` while you're at it. That crosses out trident too.

**Review by newphreak — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 1 above)
Comment (waytotheweb) [reply printer_mods/README.md]: Done for this and NozzleWiper

**Review by newphreak — APPROVED** [review]
Looks good to me

**Review by cbc02009 — APPROVED** [review]

**Comment by cbc02009** [comment]
@waytotheweb sorry, there's merged conflicts that need to be fixed. probably since I merged one of your other mods first.

**Comment by waytotheweb** [comment]
NP. I'll sort that out. The Runout Unklicky might also have the same issue...

**Comment by waytotheweb** [comment]
This should be OK now. I think Runout Unklicky should also merge now, if not let me know and I'll do another upstream merge.

**Pull request merged**